realtime-reporter/pkg/handlers/polaris.go

```diff
@@ -0,0 +1,162 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/fairwindsops/insights-plugins/plugins/admission/pkg/models"
+	"github.com/samber/lo"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/tools/cache"
+
+	fwClient "github.com/FairwindsOps/insights-plugins/realtime-reporter/pkg/client"
+	evt "github.com/FairwindsOps/insights-plugins/realtime-reporter/pkg/event"
+	"github.com/FairwindsOps/insights-plugins/realtime-reporter/pkg/polaris"
+)
+
+func PolarisHandler(resourceType string) cache.ResourceEventHandlerFuncs {
+
+	var handler cache.ResourceEventHandlerFuncs
+	handler.AddFunc = func(obj interface{}) {
```

Comment:
Please, re-check error handling in this function:
- do you really want to log and continue?
- we usually use lower-case first letter in error messages

Reply:
I suppose if we can't process the event at this point it would be reason enough to bail out--we can go with this and see if it helps catch any issues early

Reply:
Yea, usually bailing-out is what we usually want, if the process that failed is required downstream... which most of the times is the case

Reply:
Will get these changes pushed shortly